crates/driver/src/lib.rs

```diff
@@ -19,14 +19,14 @@ use url::Url;
 
 pub fn run(opts: &Options) {
     match &opts.command {
-        Command::Build => eprintln!("`fe build` doesn't work at the moment"),
+        Command::Build => tracing::error!("`fe build` doesn't work at the moment"),
         Command::Check { path, core } => {
             let mut db = DriverDataBase::default();
             let mut ingot_resolver = IngotResolver::default();
 
             let core_url = if let Some(core_path) = core {
                 if !core_path.exists() {
-                    eprintln!("the core path `{core_path}` does not exist");
+                    tracing::error!("the core path `{core_path}` does not exist");
```

Comment:
why should this be changed? it's my understanding that the user needs to set the RUST_LOG env variable to see tracing

Reply:
I set up a tracing subscriber in the Fe CLI that prints error level logs by default

Reply:
I think `eprintln` is better suited for this. tracing is meant for understanding runtime behavior and these are just messages generated by the CLI to help the user. I understand the need for tracing, but changing these messages results in what I think is cluttered output. it also doesn't seem like the LSP should be calling this function, so it shouldn't be a problem.

Reply:
Just learned about tracing's "target" field, which seems perfect for this

Reply:
I took these tracing statements out for now, will make a followup next week to explore the options